### PR TITLE
du(chore): Remove performance issues options

### PR DESCRIPTION
**Pull request by udameli**
Removing all performance issues related options and flags so that performance issues can be enabled by default. All frontend references to the feature flags will be removed in a separate PR.

Options removed:
- [x] performance.issues.all.problem-detection
- [x] performance.issues.all.problem-creation
- [x] performance-issues-ingest
- [x] performance.issues.all.early-adopter-rollout
- [x] performance.issues.all.general-availability-rollout
- [x] performance.issues.all.post-process-group-early-adopter-rollout
- [x] performance.issues.all.post-process-group-ga-rollout
- [x] organizations:performance-issues ([Flagr](https://flagr.getsentry.net/#/flags/236))
- [x] organizations:performance-issues-post-process-group

Options/Flagr feature flags to remain:
- `sentry:performance_issue_creation_rate` - granular, per-project options necessary to turn off issue creation for individual projects. needed when there are problems with issue creation for a given project
- `organizations:performance-issues-dev` ([Flagr](https://flagr.getsentry.net/#/flags/238)) - This flag is only to show UI for changing performance issue settings and won't be exposed to customers

### CODE REVIEW

**Review by wedamija — APPROVED** [review]
lgtm, I mostly skimmed the tests but the rest seems safe

**Review by gggritso — APPROVED** [review]
🚮 awesome!

**Comment by udameli** [comment]
Frontend flag removal PR to be merged first https://github.com/getsentry/sentry/pull/42556

File: src/sentry/features/__init__.py [hunk 1]
```diff
@@ -115,9 +115,6 @@
 default_manager.add("organizations:performance-anomaly-detection-ui", OrganizationFeature, True)
 default_manager.add("organizations:performance-chart-interpolation", OrganizationFeature, True)
 default_manager.add("organizations:performance-dry-run-mep", OrganizationFeature, True)
-default_manager.add("organizations:performance-issues", OrganizationFeature, True)
-default_manager.add("organizations:performance-issues-ingest", OrganizationFeature)
-default_manager.add("organizations:performance-issues-post-process-group", OrganizationFeature)
```
[the remark this answers is not in the document]
Comment (udameli) [reply src/sentry/features/__init__.py:120]: oh true! will do

**Pull request merged**